captions.html

```diff
@@ -144,7 +144,8 @@ <h2 id="others">What are the additional benefits?</h2>
       </ul>
 
       <h2 id="action">What needs to happen for this to work?</h2>
-      <p>Captions need to be made available with pre-recorded and live video content. Voice recognition software can help create captions for simple videos. Sometimes voice recognition software is built into the computer or social media platform, although often automatically-generated captions need editing. For many languages there are captioning services that are affordable.</p>
+      <p>Captions need to be available with pre-recorded and live video content.
+      As well as manual transcription, voice recognition software can help create captions but they often need editing for accuracy.</p>
```

Comment:
Not sure why you deleted all this content, which seems to be important information for newbies.